agent/engine/docker_task_engine.go

```diff
@@ -830,6 +832,16 @@ func (engine *DockerTaskEngine) Capabilities() []string {
 		}
 	}
 
+	// TODO we also need to disable this if agent already managing older tasks
+	if engine.cfg.TaskCPUMemLimit {
+		if _, ok := supportedVersions[dockerclient.Version_1_22]; ok {
+			capabilities = append(capabilities, capabilityPrefix+capabilityTaskCPUMemLimit)
```

Comment:
this shouldn't use `capabilityPrefix`. It should use `"ecs.capability."` instead (refer https://github.com/aws/amazon-ecs-agent/blob/enis/agent/app/agent_capability.go#L29)